python/tvm/contrib/hexagon/build.py

```diff
@@ -472,6 +477,15 @@ def _cleanup_directory(self):
         # Remove workspace directory on remote target
         subprocess.Popen(self._adb_device_sub_cmd + ["shell", f"rm -rf {self._workspace}"])
 
+    def _copy_artifacts_to_host(self):
+        # Copy generated artifacts from device to host
+        if self._host_workspace:
+            for file in ["tvm_rpc_android.farf", "tvm_rpc_android.log"]:
```

Comment:
The farf file only configures fastrpc to forward Hexagon logs to Android's logging facility. There's no need to pull it back off the device.

Reply:
If the intention is to pull of the debug output from Hexagon, you'll likely need to run logcat in parallel, possibly filtering out extraneous messages from the rest of the system.